homeassistant/components/redgtech/manifest.json

```
@@ -0,0 +1,12 @@
+{
+    "domain": "redgtech",
+    "name": "Redgtech",
+    "codeowners": [],
+    "documentation": "https://www.home-assistant.io/integrations/redgtech",
+    "iot_class": "cloud_polling",
+    "logo": "/brands/redgtech/logo.png",
+    "integration_type": "service",
+    "config_flow": true,
+    "quality_scale": "silver",
```

```suggestion
    "quality_scale": "bronze",
```
you won't reach anything more with this PR